logviewer/logviewer-lib.pl

```diff
@@ -27,33 +27,139 @@ sub can_edit_log
 return 0;
 }
 
-# 
-# Returns standard 
+# get_journal_since
+# Returns a list of journalctl since commands
+sub get_journal_since
+{
+return [
+        { "" => $text{'journal_since0'} },
```

Comment:
Why not return this as a 2-element array rather than a hash?

Reply:
Already did, and you liked it.

Reply:
What I mean, return it as : 

```
[ "-f", $text{'journal_since1'} ],
[ "-b", $text{'journal_since2'} ],
["-S '7 days ago'", $text{'journal_since3'} ],
```

Reply:
This is what I was trying to say, I already did!

https://github.com/webmin/webmin/blob/ca3362ee849031a8f72b4bf404c68d8517c15f0e/logviewer/logviewer-lib.pl#L30-L47